pennychat/tests/api/test_penny_chat.py

```diff
@@ -38,13 +77,34 @@ def test_penny_chat_participants_list(test_chats_1):
         assert int(user_id) in [participant['user']['id'] for participant in chat['participants']]
 
 
+@pytest.mark.django_db
+def test_penny_chat_participants_list__other_content():
+    objects = setup_test_chats()
+    chat_private_1, chat_private_2 = objects['penny_chats']
+    user1, user2, user3 = objects['users']
+    soc_prof1, soc_prof2, soc_prof3 = objects['social_profiles']
+    client = APIClient()
+    token = Token.objects.create(user=user3)
+    client.credentials(HTTP_AUTHORIZATION='Token ' + token.key)
+    user_id = user2.id
```

Comment:
nit pick - here in in the previous test I wouldn't create a `user_id` variable, it's actually easier to read `user2.id` and understand what's going on